### PR TITLE
PHP client product UUID update

### PR DESCRIPTION
**Pull request by tomglvng**

### CODE REVIEW

**Review by LevFlavien — CHANGES_REQUESTED** [review]
Getting started page has a code sample
```
$product = $client->getProductApi()->get('top');
echo $product['identifier'];
```
We should promote UUIDs instead.

TODO: Parse doc and switch Product code example to Product-UUID

**Review by LevFlavien — APPROVED** [review]

File: content/php-client/resources/products/product-drafts-uuid.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ * Returns an array like this:
+ * [
+ *     'uuid' => '12951d98-210e-4bRC-ab18-7fdgf1bd14f3',
+ *     'enabled' => true,
+ *     'family' => 'tshirt',
+ *     'categories' => ['summer_collection'],
+ *     'groups' => [],
+ *     'values' => [
+ *         'name' => [
+ *              [
```
Comment (pierallard) [inline content/php-client/resources/products/product-drafts-uuid.md:23]: You should have a "sku" value here.

File: content/php-client/resources/products/products-uuid.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+ * Returns an array like this:
+ * [
+ *     'uuid' => '12951d98-210e-4bRC-ab18-7fdgf1bd14f3',
+ *     'enabled' => true,
+ *     'family' => 'tshirt',
+ *     'categories' => ['summer_collection'],
+ *     'groups' => [],
+ *     'parent' => null,
+ *     'values' => [
+ *         'name' => [
```
Comment (pierallard) [inline content/php-client/resources/products/products-uuid.md:23]: Same

File: content/php-client/resources/products/products-uuid.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+// set the limit of 50 products per page, calculate the total number of researched products, apply a research
+$firstPage = $client->getProductUuidApi()->listPerPage(50, true, ['search' => $searchFilters, 'scope' => 'ecommerce']);
+```
+
+::: warning
+There is a maximum limit allowed on server side for the parameter `limit`.
+:::
+
+::: warning
+Setting the parameter `with_count`  to `true`  can drastically decrease the performance.
```
Comment (pierallard) [inline content/php-client/resources/products/products-uuid.md:111]:
```suggestion
Setting the parameter `with_count`  to `true` can drastically decrease the performance.  
```

File: content/php-client/resources/products/products-uuid.md [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+
+$client->getProductUuidApi()->create('844c736b-a19b-48a6-a354-6056044729f0', [
+    'uuid' => '844c736b-a19b-48a6-a354-6056044729f0',
+    'enabled' => true,
+    'family' => 'tshirt',
+    'categories' => ['summer_collection'],
+    'groups' => [],
+    'parent'=> null,
+    'values' => [
+        'name' => [
```
Comment (pierallard) [inline content/php-client/resources/products/products-uuid.md:164]: same

File: content/php-client/resources/products/products-uuid.md [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+
+$client->getProductUuidApi()->upsert('844c736b-a19b-48a6-a354-6056044729f0', [
+    'uuid' => '844c736b-a19b-48a6-a354-6056044729f0',
+    'enabled' => true,
+    'family' => 'tshirt',
+    'categories' => ['summer_collection'],
+    'groups' => [],
+    'parent'=> null,
+    'values' => [
+        'name' => [
```
Comment (pierallard) [inline content/php-client/resources/products/products-uuid.md:214]: etc

**Review by pierallard — APPROVED** [review]

File: content/php-client/resources/products/products-uuid.md (hunk 5 above)
Comment (LevFlavien) [reply content/php-client/resources/products/products-uuid.md:235]: ty

**Pull request merged**